src/mixer/audio.ts

```diff
@@ -323,25 +340,64 @@ export class AudioEngine extends ((EngineEmitter as unknown) as {
     this.micMixGain.gain.value = value;
   }
 
-  getLevel(source: LevelsSource) {
+  getLevels(source: LevelsSource, stereo: boolean) {
+    let analysisBuffer = new Float32Array(ANALYSIS_FFT_SIZE);
+    let analysisBuffer2 = new Float32Array(ANALYSIS_FFT_SIZE);
```

Comment:
I'd prefer if we didn't allocate this buffer if we don't need stereo data. (On that note, is it maybe worth reusing this buffer instead of re-allocating every frame? It should get GC'd but that might add pressure.)

Reply:
Fixed. Also made it smaller too 👍